### DISCUSSION TITLE
服务端有多个公网IP，如何实现多wan访问？

### DISCUSSION THREAD

**Discussion by microka** [AlistGo/alist#6457] [General]
在 Windows 11 Pro 23H2 (22631.3527) 部署了 AList 3.34.0，路由器 OpenWRT 拨号，wan1 wan2 均有 IPv4 公网 IP，路由策略默认走 wan1，通过 wan1 和 wan2 IP 均能访问该 Windows 的 RDP 以及 qBittorrent 服务。
现在从外部网络能通过 wan1 IP 访问 AList，但无法通过 wan2 IP 访问。AList 监听地址 0.0.0.0，本地网卡 IPv4 地址为 192.168.xx.xx。
请问能否通过修改 AList 监听逻辑（或其它方法）使之也能通过 wan2 IP 访问呢？感谢~

**Comment by hahach** [comment]
你OP防火墙的端口转发是不是没设置好

**Reply by microka** [reply]
端口转发是没问题的，毕竟 wan1 也是端口转发访问的。
